### DISCUSSION TITLE
Set CustomModelData on item NBT

### DISCUSSION THREAD

**Discussion by ryleu** [Minestom/Minestom#1347] [Help] [answered]
How can I set the CustomModelData NBT field on an item when giving it to a player?

**Comment by GreatWyrm** [comment]
You can use the `CUSTOM_MODEL_DATA` Tag field in the ItemTags class.

**Comment by DevBlokJochem** [comment, marked as answer]
You can apply the custom model data on building the item.
```
ItemStack.builder(Materials.DIRT).meta(meta -> meta.customModelData(1)).build();
```
